ui/components/app/snaps/snap-ui-renderer/snap-ui-renderer.test.js

```diff
@@ -132,41 +27,6 @@ describe('SnapUIRenderer', () => {
     expect(container).toMatchSnapshot();
   });
 
-  it('renders footers', () => {
```

Comment:
IMO we should keep the basic tests for footers and interactive inputs in this file as they are basic tests for the system and have impacts on the renderer/context directly.